src/router/routes/landing-page.js

```diff
@@ -0,0 +1,19 @@
+import signInRoutes from "./signIn"
+import generateAccountRoutes from "./generateAccount"
+
+const landingPageRoutes = [{
+  path: "/landing-page",
+  component: () => import(/* webpackChunkName */ "../../views/LandingPage/Layout"),
+  children: [
+    {
+      path: "/landing-page",
+      name: "landing-page",
+      component: () => import(/* webpackChunkName */ "../../views/LandingPage")
```

Comment:
Hi @ciniiia, can you please also change these like others using `@`

Reply:
done . thank you!

Reply:
done . thank you!